www/src/components/Toc.scss

```diff
@@ -34,5 +34,10 @@
       font-weight: 500;
       color: $primary-500;
     }
+
+    &.active {
```

Comment:
this look like the same styles as for `hover` and `focus` states right above (line 33), you can just add this selector to them instead of duplicating styles, i.e.
```
&:hover,
&:focus,
&.active {
  font-weight: 500;
  color: $primary-500;
}
```